cluacov-scm-1.rockspec

```diff
@@ -13,7 +13,7 @@ number of lines incorrectly marked as missed. Depends on LuaCov.
    license = "MIT"
 }
 dependencies = {
-   "lua >= 5.1, < 5.4",
+   "lua >= 5.1",
```

Comment:
I think the maximum version was suggested by @hishamhm  ?

Reply:
Since this is the SCM rockspec, I think the limit should not be there. In the versioned ones, it should be there.